testing/tests/image.lua

```diff
@@ -78,12 +83,23 @@ love.test.image.ImageData = function(test)
   local r2, g2, b2 = idata:getPixel(25, 25)
   test:assertEquals(1, r2+g2+b2, 'check set to red')
 
-  -- check encoding to an image
+  -- check encoding to an image (png)
   idata:encode('png', 'test-encode.png')
-  local read = love.filesystem.openFile('test-encode.png', 'r')
-  test:assertNotNil(read)
+  local read1 = love.filesystem.openFile('test-encode.png', 'r')
+  test:assertNotNil(read1)
   love.filesystem.remove('test-encode.png')
 
+  -- check encoding to an image (exr)
+  idata:encode('png', 'test-encode.exr')
```

Comment:
The format here should be `exr`, otherwise it's just encoding a png and then saving it to a file with an .exr extension I think.

But exr only supports floating point and uint formats.